decisions/tracking.md

```diff
@@ -0,0 +1,28 @@
+# Support for Tracking
+
+Type-ramp system has following values for tracking
+
+font size | letter Spacing
```

Comment:
We should mention that this is the letter spacing as defined in Sketch. We could also calcualted the em tracking which is what Android uses and add it as a column. Tools like lillustrator/photoshop use thousands of em which can also be calculated.